.github/workflows/rust.yaml

```diff
@@ -59,9 +68,17 @@ jobs:
     strategy:
       matrix:
         os:
-          - ${{ github.repository_owner == 'subspace' && 'self-hosted-ubuntu-2204-amd64-base' || 'ubuntu-22.04' }}
+          - ${{ 
+                github.repository_owner == 'subspace' 
+                  && 'self-hosted-ubuntu-2204-amd64-base' 
+                  || 'ubuntu-22.04' 
+            }}
           - macos-12
-          - ${{ github.repository_owner == 'subspace' && 'self-hosted-wnd-ms-servier-2019-10-x64-based' || 'windows-2022' }}
+          - ${{ 
+                github.repository_owner == 'subspace' 
+                  && 'self-hosted-wnd-ms-servier-2019-10-x64-based' 
```

Comment:
Naming here is weird and inconsistent. Ubuntu ends with `-base` (will we have non-base version?) and Windows ends with `10-x64-based` for some reason. If keeping it consistent then it should be `self-hosted-windows-server-2019-base`, ideally without `-base` though.